behaviors/wysiwyg.js

```diff
@@ -13,6 +13,69 @@ var _ = require('lodash'),
   refAttr = references.referenceAttribute;
 
 /**
+ * whan that sellection, with his ranges soote
+ * the finalle node hath perced to the roote
+ * @param {Node} node
+ */
+function selectAfter(node) {
+  var range = document.createRange(),
+    selection = window.getSelection();
+
+  range.setStartAfter(node); // set the caret after this node
+  range.collapse(true);
+  selection.removeAllRanges();
+  selection.addRange(range);
+  // Of the DOM, to inputte's end they wende
+  // the hooly blisful caret for to seke
+  // that hem hath holpen, when that they were pasted
+}
+
+/**
+ * split innerHTML into paragraphs
+ * @param {string} str
+ * @returns {array}
+ */
+function splitParagraphs(str) {
+  return str.split('¶');
```

Comment:
If they type that character, everything's fine. If they paste in html that contains that character, it'll make a new paragraph. Hmmmmm